### PR TITLE
Task#4 integration

### PR DESCRIPTION
**Pull request by BaHGet**

### CODE REVIEW

**Comment by BaHGet** [comment]
by viewing these changes on the Production branch it shows that:
it's working properly 
pull request reviewed with approval.

**Comment by BaHGet** [comment]
by viewing these changes on the Production branch it shows that:
it's working properly
pull request reviewed with approval.

**Comment by BaHGet** [comment]
by viewing these changes on the Production branch it shows that:
it's working properly
pull request reviewed with approval.

**Review by BaHGet — COMMENTED** [review]
by viewing these changes on the Production branch it shows that:
it's working properly
pull request reviewed with approval.

**Review by ZeyadM7MD — APPROVED** [review]
by viewing these changes on the Production branch it shows that:
it's working properly
pull request reviewed with approval.

File: index.html [hunk 1]
```diff
@@ -12,18 +12,82 @@
...
+            <div class="silder-btn">
+                <span class="left-arrow"></span>
+                <h1>Pick your coffee</h1>
+                <span class="right-arrow"></span>
+            </div>
+
+            <div class="cards">
+                <div class="card">
+                    <div>
+                        <img src="./deyof-assets/pick-your-coffee/Espresso.png" alt="Espresso">
```
Comment (BaHGet) [inline index.html:40]: Should be assigned by js resolver func

**Pull request merged**